numprocs=10
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
Traceback (most recent call last):
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/./run_model.py", line 68, in <module>
    c = Config(args.config_file)
  File "/home/people/22213094/CBG_Model_Fleming_PTS/Cortex_BasalGanglia_DBS_model/config.py", line 86, in __init__
    conf = yaml.safe_load(f)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/opt/software/python/3.9.15-with-shared/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 1
could not find expected ':'
  in "/home/people/22213094/CBG_Model_Fleming_PTS/configs/pts_test_48.yml", line 17, column 2
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[58599,1],4]
  Exit code:    1
--------------------------------------------------------------------------
